hbase_storage/src/main/java/com/splicemachine/access/client/SkeletonClientSideRegionScanner.java

```diff
@@ -155,29 +155,29 @@ public boolean nextRaw(List<Cell> result) throws IOException {
      * created by MemStore flushes or current scanner fails due to compaction
      */
     public void updateScanner() throws IOException {
-            if (LOG.isDebugEnabled()) {
-                SpliceLogUtils.debug(LOG,
-                        "updateScanner with hregionInfo=%s, tableName=%s, rootDir=%s, scan=%s",
-                        hri, htd.getNameAsString(), rootDir, scan);
-            }
-            if (flushed) {
+        if (LOG.isDebugEnabled()) {
+            SpliceLogUtils.debug(LOG,
+                    "updateScanner with hregionInfo=%s, tableName=%s, rootDir=%s, scan=%s",
+                    hri, htd.getNameAsString(), rootDir, scan);
+        }
+        if (flushed) {
+            if (LOG.isDebugEnabled())
+                SpliceLogUtils.debug(LOG, "Flush occurred");
+            if (this.topCell != null) {
                 if (LOG.isDebugEnabled())
-                    SpliceLogUtils.debug(LOG, "Flush occurred");
-                if (this.topCell != null) {
-                    if (LOG.isDebugEnabled())
-                        SpliceLogUtils.debug(LOG, "setting start row to %s", topCell);
-                    //noinspection deprecation
-                    scan.setStartRow(Bytes.add(CellUtil.cloneRow(topCell), new byte[]{0}));
-                }
+                    SpliceLogUtils.debug(LOG, "setting start row to %s", topCell);
+                //noinspection deprecation
+                scan.setStartRow(Bytes.add(CellUtil.cloneRow(topCell), Bytes.toBytes(0)));
```

Comment:
This is not correct, this appends 4 NULL bytes instead of a single NULL byte.

Reply:
Done